### PR TITLE
Add `tauri::Builder` doc comment examples

### PR DESCRIPTION
**Pull request by probablykasper**
### What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [x] Docs
- [ ] New Binding issue #___
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

### Does this PR introduce a breaking change?

- [ ] Yes, and the changes were approved in issue #___
- [x] No

### Checklist
- [x] When resolving issues, they are referenced in the PR's title (e.g `fix #___, #___`)
- [x] A change file is added if any packages will require a version bump due to this PR per [the instructions in the readme](https://github.com/tauri-apps/tauri/blob/dev/.changes/readme.md).
- [x] I have added a convincing reason for adding this feature, if necessary

### CODE REVIEW

File: core/tauri/src/app.rs [hunk 1]
```diff
@@ -651,6 +660,15 @@ impl<R: Runtime> Builder<R> {
   }
 
   /// Defines the JS message handler callback.
+  ///
+  /// # Example
+  /// ```rust,ignore
+  /// tauri::Builder::default()
+  ///   .invoke_handler(tauri::generate_handler![
+  ///     command_1,
```
Comment (lucasfernog) [inline core/tauri/src/app.rs:679]: maybe also add the commands to this example? just to show that it must use #[tauri::command]

File: core/tauri/src/app.rs [hunk 2]
```diff
@@ -818,12 +859,56 @@ impl<R: Runtime> Builder<R> {
   }
 
   /// Sets the menu to use on all windows.
+  ///
+  /// # Example
+  /// ```rust,ignore
+  /// use tuari::{MenuEntry, Submenu, MenuItem, Menu, CustomMenuItem};
```
Comment (lucasfernog) [inline core/tauri/src/app.rs]:
```suggestion
  /// use tauri::{MenuEntry, Submenu, MenuItem, Menu, CustomMenuItem};
```

File: core/tauri/src/app.rs (hunk 1 above)
Comment (probablykasper) [reply core/tauri/src/app.rs:679]: Good call, added it!

**Review by lucasfernog — APPROVED** [review]
Amazing PR, code examples helps us find mistakes in our API :)

**Pull request merged**